src/scripts/pbs_submit.sh

```diff
@@ -154,27 +154,37 @@ bls_set_up_local_and_extra_args
 # handle queue overriding
 [ -z "$bls_opt_queue" ] || grep -q "^#PBS -q" $bls_tmp_file || echo "#PBS -q $bls_opt_queue" >> $bls_tmp_file
 
-# Extended support for MPI attributes
+# Extended support for MPI attributes (+GPU for pbs_sched)
+pmaui=$(/usr/bin/pgrep maui)
```

Comment:
this seems like a terrible idea

Reply:
I mean, a more exact process name match, at least...

Reply:
Anyway, just seems real easy for any old user to fool this script by running a program named maui.  If that's really the way we have to do it, at least consider adding the `-x` option to pgrep to get an exact process name match, although even that still feels pretty sketchy to me.

Reply:
I'll add a pbs_maui parameter to the blah config file (like the existing pbs_pro parameter), defaulting to "no".